server/src/repositories/album.repository.ts

```diff
@@ -220,8 +220,10 @@ export class AlbumRepository {
     await this.db.deleteFrom('albums').where('ownerId', '=', userId).execute();
   }
 
-  async removeAsset(assetId: string): Promise<void> {
-    await this.db.deleteFrom('albums_assets_assets').where('albums_assets_assets.assetsId', '=', assetId).execute();
+  @GenerateSql({ params: [[DummyValue.UUID]] })
```

Comment:
Per Mert's recent PR (#18301) this should probably be removed as well.

Reply:
I don't think we need to since that uses a dummy uuid which won't ever match anything.